.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,52 @@
+name: Conda-based pytest
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [2.7, 3.6, 3.9]
+        slycot: ["", "conda"]
+        array-and-matrix: [0]
+        include:
+          - python-version: 3.9
+            slycot: conda
+            array-and-matrix: 1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+    - name: Set up conda
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH
+        conda create -q -n test-environment python=${{matrix.python-version}} 
+    - name: Install dependencies
+      run: |
+        source $CONDA/bin/activate test-environment
+        conda install pip coverage pytest
+        conda install numpy matplotlib scipy
+        if [[ '${{matrix.python-version}}' == '2.7' ]]; then
+          # matplotlib for Python 2.7 seems to require X11 to run correctly
+          sudo apt install -y xvfb
+        fi
```

Comment:
Why still bother with Python2? Time to ditch it for the next release.

Also, pytest-xvfb is the way to go.

Reply:
Thanks for the suggestion.  Will try shifting to pytest-xvfb.  I figure we may as well test against Python 2.7 for as long as it continues to work.  Amazingly, it is still the default Python on MacOS 11 (Big Sur).

Reply:
If we don't actively remove it, every PR with a failing Py2 test will try to workaround it. I think we should save contributors from this torture.